Python/ceval.c

```diff
@@ -4180,7 +4181,7 @@ _PyEval_EvalCode(PyThreadState *tstate,
         if (keyword == NULL || !PyUnicode_Check(keyword)) {
             _PyErr_Format(tstate, PyExc_TypeError,
                           "%U() keywords must be strings",
-                          co->co_name);
+                          qualname);
```

Comment:
Are there any suggestions for how to test this from Python? I couldn't figure out how without getting a ``SyntaxError``.

Reply:
Using the ** should do the trick.
For example:

`>>> func = lambda *args, **kwargs: None
>>> kwargs = {None: None}
>>> func(kwargs)
>>> func(*kwargs)
>>> func(**kwargs)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: <lambda>() keywords must be strings
`

Reply:
Unfortunately it looks like this behavior changed a bit in the newest releases so that that code isn't actually reached:
```
PS > py -3.7 -c "f = lambda x: None; f(**{1:1})"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: <lambda>() keywords must be strings
PS > py -3.8 -c "f = lambda x: None; f(**{1:1})"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: <lambda>() keywords must be strings
PS > py -3.9 -c "f = lambda x: None; f(**{1:1})"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: keywords must be strings
```

Reply:
Oh I see... Probably related to https://github.com/python/cpython/commit/0567786d26348aa7eaf0ab1b5d038fdabe409d92 .